api/configuration/index.ts

```diff
@@ -34,14 +34,35 @@ export const hasConfigValue = reference => {
   return config.has(reference)
 }
 
-export const getIDamSecret = (secretsConfig): string => {
-  const ERROR_IDAM_SECRET_NOT_FOUND =
-    'mc-idam-client-secret not found on this environment.'
-  if (propsExist(secretsConfig, ['secrets', 'rpx', 'mc-idam-client-secret'])) {
-    // tslint:disable-next-line: no-string-literal
-    return secretsConfig['secrets']['rpx']['mc-idam-client-secret']
-  } else {
-    console.log(ERROR_IDAM_SECRET_NOT_FOUND)
-    return ''
-  }
-}
+/**
+ * Allows us to integrate the Azure key-vault flex volume, so that we are able to access Node configuration values.
+ */
+propertiesVolume.addTo(config, { mountPoint: '/Volumes/mnt/secrets/'})
```

Comment:
please remove the `mountPoint` otherwise this will fail in AKS

Reply:
Done